src/test/kotlin/no/nav/syfo/TestData.kt

```diff
@@ -30,7 +30,7 @@ import no.nav.syfo.sm.Diagnosekoder
 val behandletTidspunkt = LocalDateTime.of(2019, 1, 1, 0, 0)
 val signaturDato = LocalDateTime.of(2019, 1, 1, 0, 0)
 
-fun Diagnosekoder.DiagnosekodeType.toDiagnose() = Diagnose(system = oid, kode = code)
+fun Diagnosekoder.DiagnosekodeType.toDiagnose() = Diagnose(system = oid, kode = code, tekst = text)
```

Comment:
Har alle diagnoser en tekst nå, eller er denne valgfri? For papirsykmeldingene er det jo ganske minimalt hva vi mottar..

Reply:
Jepp alle diagnoser har ein tekst.

Reply:
Alle kommer til å ha ein tekst, dersom dei ikkje har det blir det "", dette fikser eg i syfosmpapirmottaket